nav2_bt_navigator/behavior_trees/follow_point.xml

```diff
@@ -0,0 +1,20 @@
+<!--
```

Comment:
Add info about this in the README section for BT.XML files https://github.com/ros-planning/navigation2/tree/f4685e0786608db939f8c7e32815639268556b02/nav2_bt_navigator#navigation-behavior-trees

Reply:
Done!!